### PR TITLE
Surface albedo test

### PR DESCRIPTION
**Pull request by tyfolino**
Intention: I wanted to add a test using the calculate surface albedo feedback function.

Actions: Added test to test_frontend.py. Verified it passes with pytest. Linted and formatted!

### CODE REVIEW

File: climkern/tests/test_frontend.py [hunk 1]
```diff
@@ -1,14 +1,23 @@
-from climkern.frontend import *
 import xarray as xr
 
-ctrl, pert = tutorial_data('ctrl'),tutorial_data('pert')
+from climkern.frontend import *
+
+ctrl, pert = tutorial_data("ctrl"), tutorial_data("pert")
```
Comment (emilygrabowski) [inline climkern/tests/test_frontend.py:5]: Consider adding definitions (here or in the documentation) about any different tutorial data.

File: climkern/tests/test_frontend.py [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+
 
 def test_calc_T_feedbacks() -> None:
```
Comment (emilygrabowski) [inline climkern/tests/test_frontend.py:8]: Yay a test! Looks good (and bonus for typing)

File: climkern/tests/test_frontend.py [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+        ctrl.T, ctrl.TS, ctrl.PS, pert.T, pert.TS, pert.PS, pert.TROP_P, kern="GFDL"
+    )
+    dTS_glob_avg = spat_avg(pert.TS - ctrl.TS)
+    LR_val = (spat_avg(LR) / dTS_glob_avg).mean()
+    Planck_val = (spat_avg(Planck) / dTS_glob_avg).mean()
+    xr.testing.assert_allclose(LR_val, xr.DataArray(-0.41), atol=0.01)
+    xr.testing.assert_allclose(Planck_val, xr.DataArray(-3.12), atol=0.01)
+
+
+def test_albedo_feedbacks() -> None:
```
Comment (emilygrabowski) [inline climkern/tests/test_frontend.py:19]: I don't know the science, so I will trust that the test is working :D

**Review by emilygrabowski — APPROVED** [review]
Looks good! Maybe add some documentation once you have the tests finished.

File: climkern/tests/test_frontend.py (hunk 1 above)
Comment (tyfolino) [reply climkern/tests/test_frontend.py:5]: You're right, I'll add some documentation to it. Thanks!

**Pull request merged**